deploy/Dockerfile

```
@@ -0,0 +1,45 @@
+FROM adoptopenjdk:11
```

```suggestion
FROM eclipse-temurin:11
```

Let's use our own stuff